assets/src/editor/events/tickets/editor-ticket/grid-view/editor-ticket-entities-grid-view.js

```diff
@@ -26,18 +27,20 @@ const EditorTicketEntitiesGridView = ( {
 	htmlClass,
 	...otherProps
 } ) => {
-	htmlClass = htmlClass ?
+	const classes = htmlClass ?
```

Comment:
nice catch :+1:

Reply:
might take me awhile to break that habit : \